jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/CreateQueueDialog.kt

```diff
@@ -53,7 +54,7 @@ class CreateQueueDialog(
         launch {
             try {
                 createQueue()
-                runInEdt(ModalityState.any()) {
+                withContext(getCoroutineUiContext(ModalityState.any())) {
```

Comment:
should we do a wholesale replacement of `runInEdt` ?

Reply:
I guess maybe not - the comment here was that it was already in a coroutine context.

Reply:
Yeah in a non-coroutine context we should probably not use `runBlocking { withContext(getCoroutineUiContext)` over `runInEdt`. we should probably replace our `runInEdtAndWait`'s though since it's marked `@TestOnly`